docs/delivery_methods/ios.md

```diff
@@ -22,7 +22,7 @@ https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_
 class CommentNotification
   deliver_by :ios do |config|
     config.device_tokens = ->(recipient) { recipient.notification_tokens.where(platform: :iOS).pluck(:token) }
-    config.format = ->(apn) {
```

Comment:
Since we're using the Apnotic gem and not an HTTP request, I figured this shouldn't be renamed to json. I copied the line in the UPGRADE guide on accident.

Reply:
Yep, that makes sense. I'll revert that change and update the docs accordingly